spec/models/day_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Day, type: :model do
   describe 'validations' do
     it { should validate_presence_of(:week) }
-    it { should validate_presence_of(:day_of_week) }
+    it { should define_enum_for(:day_of_week) }
```

Comment:
You can still leave that old line of code in here (for validating presence)
You want to check the enum _in addition_ to validating presence

Reply:
(validate_presence_of means a day HAS TO have this attribute, otherwise it won't save to the database. define_enum_for is just checking that attribute is, in fact, an enum)

Reply:
ALSO for `define_enum_for`, you want to check what it's actually defining the enum for. Here's an example from my group's Monster Shop: 
`it {should define_enum_for(:role).with_values([:regular, :merchant, :admin])}
`